recipes/vir-simd/recipe.yaml

```diff
@@ -0,0 +1,56 @@
+context:
+  name: vir-simd
+  version: "0.4.4"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/mattkretz/vir-simd/archive/v${{ version }}.tar.gz
+  sha256: 0d2953e1219798967b156aa392de6430099d85887770e5c81aefe8fb63e9464c
+
+build:
+  # TODO: does this work here? vir-simd is a C++, header-only library
+  # noarch: python
```

Comment:
For review: this is a C++ header-only library.

Ideally, I would like to install it via CMake, because this provides me a package config for `find_package` in CMake downstream, but I think that will make it then platform dependent because of boilerplates in the CMake configs... Anyway, would also be ok to make it arch-dependent.